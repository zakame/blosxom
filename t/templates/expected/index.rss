Content-Type: text/xml; charset=UTF-8

<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0"
    xmlns:dc="http://purl.org/dc/elements/1.1/"
    xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
    xmlns:atom="http://www.w3.org/2005/Atom">

    <channel>
        <title>My Weblog</title>
        <link>http://localhost/</link>
        <description>Yet another Blosxom weblog.</description>
        <managingEditor>author@example.com (A. U. Thor)</managingEditor>
        <webMaster>author@example.com (A. U. Thor)</webMaster>
        <pubDate>Sun, 11 Nov 2007 11:11:00 +0000</pubDate>
        <language>en</language>
        <generator>blosxom 2.1.0</generator>
        <atom:link href="http://localhost/index.rss" rel="self" type="application/rss+xml" />
        <sy:updatePeriod>hourly</sy:updatePeriod>
        <sy:updateFrequency>1</sy:updateFrequency>
        <sy:updateBase>2000-01-01T12:00+00:00</sy:updateBase>


        <item>
            <title>Lorem ipsum II</title>
            <link>http://localhost/web/2.html</link>
            <guid isPermaLink="true">http://localhost/web/2.html</guid>
            <pubDate>Sun, 11 Nov 2007 11:11:00 +0000</pubDate>
            <description>
Lorem ipsum dolor sit amet ipso facto.Lorem ipsum dolor sit amet ipso
facto.Lorem ipsum dolor sit amet ipso facto. Lorem ipsum dolor sit
amet ipso facto.Lorem ipsum dolor sit amet ipso facto. 

Lorem ipso hoccer foer tyr wuz de ras cora dola tymon ipso hoc cer fuz
de ramas cora dola tymon ipso hoccer. Lorem ipso foer tyr wuz de ramas
cora dola tym ipso hor.

            </description>
        </item>

        <item>
            <title>Lorem ipsum</title>
            <link>http://localhost/1.html</link>
            <guid isPermaLink="true">http://localhost/1.html</guid>
            <pubDate>Wed, 19 Jul 2006 22:54:00 +0000</pubDate>
            <description>Lorem ipsum dolor sit amet ipso facto.Lorem ipsum dolor sit amet ipso
facto.Lorem ipsum dolor sit amet ipso facto. Lorem ipsum dolor sit
amet ipso facto.Lorem ipsum dolor sit amet ipso facto. 

facto.Lorem ipsum dolor sit amet ipso facto.Lorem ipsum dolor sit amet
ipso facto.Lorem ipsum dolor sit amet ipso facto. Lorem ipsum dolor
sit amet ipso facto.Lorem ipsum dolor sit amet ipso facto. 

Lorem ipso hoccer foer tyr wuz de ras cora dola tymon ipso hoc cer fuz
de ramas cora dola tymon ipso hoccer. Lorem ipso foer tyr wuz de ramas
cora dola tym ipso hor.Lorem ipso foer tyr wuz de ramas cora dola tym
ipso hor tyr wuz de ramas cora dola tymon ipso foer tyr wuz de ramas
cora dola tymo. Lorem ipso hoccer foer tyr wuz de ras cora dola tymon
ipso hoc cer fuz de ramas cora dola tymon ipso hoccer. Lorem ipso foer
tyr wuz de ramas cora dola tym ipso hor.
            </description>
        </item>

    </channel>
</rss>
