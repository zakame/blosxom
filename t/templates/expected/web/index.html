Content-Type: text/xhtml; charset=UTF-8

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
          "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html>
    <head>
        <link rel="alternate" type="application/rss+xml" title="RSS" href="http://localhost/index.rss" />
        <title>My Weblog   
        </title>
    </head>
    <body>
        <h1>My Weblog</h1>
        <h2>  </h2>
        <h3>2007-11-11</h3>
        <h4><a name="2">Lorem ipsum II</a></h4>
        <div>
Lorem ipsum dolor sit amet ipso facto.Lorem ipsum dolor sit amet ipso
facto.Lorem ipsum dolor sit amet ipso facto. Lorem ipsum dolor sit
amet ipso facto.Lorem ipsum dolor sit amet ipso facto. 

Lorem ipso hoccer foer tyr wuz de ras cora dola tymon ipso hoc cer fuz
de ramas cora dola tymon ipso hoccer. Lorem ipso foer tyr wuz de ramas
cora dola tym ipso hor.
</div>
        <p>posted at: 11:11 | path: <a href="http://localhost/web">/web </a> | <a href="http://localhost/2007/11/11#2">permanent link to this entry</a></p>

    </body>
</html>
